script/tasks/updateFacetConfig.sh

```
@@ -42,64 +35,59 @@ updateFacetConfig() {
+    # ensure all required .env values are set
+    checkRequiredVariablesInDotEnv $NETWORK
+
+    
+    # repeatedly call selected script until it's succeeded or out of attempts
+    ATTEMPTS=1
+    while [ $ATTEMPTS -le "$MAX_ATTEMPTS_PER_SCRIPT_EXECUTION" ]; do
+      echo "[info] now executing $SCRIPT on $DIAMOND_CONTRACT_NAME in $ENVIRONMENT environment on $NETWORK (FILE_SUFFIX=$FILE_SUFFIX, USE_MUTABLE_DIAMOND=$USE_MUTABLE_DIAMOND)"
+
+      if [[ "$DEBUG" == *"true"* ]]; then
+        # print output to console
+        RAW_RETURN_DATA=$(NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT") forge script "$SCRIPT_PATH" -f $NETWORK -vvvv --json --silent --broadcast --skip-simulation --legacy)
```

Shellcheck

[warning] This assignment is only seen by the forked process.

(SC2097)